### ISSUE TITLE
Travis CI runs for plain master branch on pull requests

### ISSUE THREAD

**Issue by filfreire** [yegor256/rultor#1240]
The execution of a given PR seems to be running from plain `master` and not from PR code. Example, the PR #1239 (to bump copyright dates), fails with this log https://travis-ci.org/yegor256/rultor/builds/324258183

/cc @yegor256

**Comment by 0crat** [comment]
@yegor256 please, pay attention to this issue

**Comment by yegor256** [comment]
@filfreire it's not possible. Rultor works for three years with hundreds of projects. Are you sure??

**Comment by filfreire** [comment]
@yegor256 yes I'm sure. This is only related to the build Travis makes everytime someone submits a PR or pushes a commit to a PR.

It doesn't make sense that in the PR to fix all the copyright dates, that Travis, inside that PR, failed at a given point because of the same dates as if the changes weren't applied.

See on Travis the branch that the execution says it's running at:
![image](https://user-images.githubusercontent.com/11976836/35018276-b02ede86-fb18-11e7-9576-9972e9605064.png)

**Comment by yegor256** [comment]
@filfreire wait, what Travis has to do with us? Report to them )
